### ISSUE TITLE
Incompatible with newer version of Audacity

### ISSUE THREAD

**Issue by shyboss2001** [sinshu/freeverb#1]
Dear sinshu (aka Jezar at Dreampoint),

The Freeverb VST Plugin does not work properly on newer versions of Audacity (Example: 3.6.1 and 3.6.2). Can you make it compatible with latest version of Audcacity.
![IncompatiblePlugin](https://github.com/user-attachments/assets/195773d0-42ca-42da-9bd9-fb87e9350492)

**Comment by sinshu** [comment]
Thank you for the report, but I apologize for the confusion. I am not the author of Freeverb. This repository contains code that I copied from somewhere on the internet to study reverb algorithms (though I no longer remember where exactly I copied it from), and I have no intention of continuing the development of Freeverb. Additionally, I have no experience building VSTs from code, so it’s difficult for me to provide advice on the issue of Audacity not recognizing the VST...
